Composer/packages/extensions/adaptive-form/src/components/fields/ObjectField.tsx

```diff
@@ -7,7 +7,7 @@ import { getOrderedProperties } from '../../utils';
 import { FormRow } from '../FormRow';
 
 const ObjectField: React.FC<FieldProps<object>> = function ObjectField(props) {
-  const { schema, uiOptions, depth, value, ...rest } = props;
+  const { schema, uiOptions, depth, value, label, ...rest } = props;
```

Comment:
Can you explain this? It doesn't look like it's being used which makes me think we just want to exclude it from `rest`. Is that right?

If so, this may be risky without doing a regression pass on the property editor.

Reply:
When you set the top-level `label` property to `false` in the form schema, that `label` value gets passed to every schema field in the object since the `label` value is included in `rest`. Subsequently, every field label will be hidden unintentionally. By including `label` when we deconstruct `props` it is no longer included in `rest`, and the labels appear as expected.

Reply:
That makes sense. And we rely on the SchemaField to get a new label for the property anyways.